### PR TITLE
Update telemetry_fields_VAV.yaml

### PR DESCRIPTION
**Pull request by RathishKJ**
Hello,

Hereby raising the pull request for the VAVs

-zone_air_occupied_cooling_temperature_setpoint
-zone_air_unoccupied_cooling_temperature_setpoint 

regards,
Rathish

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -4733,3 +4733,13 @@ HWRWISOVPM:
...
+
+CSOUC:
+  description: "Cooling occupied/unoccupied setpoint control."
+  id: "292411e6ab73486ca5bc84874ad47aaa"
+  is_abstract: true
+  uses:
+  - zone_air_unoccupied_cooling_temperature_setpoint
+  - zone_air_occupied_cooling_temperature_setpoint
+  implements:
+  - CONTROL
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:4745]: Include the zone_air_temperature_sensor here, and change from CONTROL to OPERATIONAL. You did not include the actuator (e.g. valve or damper) that actually controls the zone to temperature.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 2]
```diff
@@ -471,4 +471,18 @@ VAV_RD_CO2C:
...
+########################
+
+VAV_SD_DSP_CSOUC:
+  id: "16129160067017605120"
+  description: "Single duct dual set point VAV with cooling control."
+  is_canonical: true
+  implements:
+  - VAV
+  - SD
+  - DSP
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:487]: This device will have all of these fields:
- zone_air_temperature_sensor
- zone_air_cooling_temperature_setpoint
- zone_air_heating_temperature_setpoint
- zone_air_unoccupied_cooling_temperature_setpoint
- zone_air_occupied_cooling_temperature_setpoint

Where are the heating setpoints? And why are the unoccupied/occupied setpoints needed if the effective setpoints are already on the device under DSP?

**Review by tasodorff — CHANGES_REQUESTED** [review]

**Comment by tasodorff** [comment]
@RathishKJ please respond to comments and make changes as necessary.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -36,6 +36,16 @@ SD:
   - supply_air_damper_percentage_command
   implements:
   - CONTROL
+  
+ZA:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:40]: this is already defined. why are you redefining it?

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 4]
```diff
@@ -885,6 +895,17 @@ MINFC:
...
+  
+MAXFC:
+  id: "16039155144679489536"
+  description: "Maximum flow control for entire loop."
+  is_abstract: true
+  implements:
+  - CONTROL
+  uses:
+  - bypass_valve_percentage_command
+  - max_flowrate_setpoint
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:907]: unless you are aggregating a bunch of setpoints, this is not a `max_flowrate_setpoint` -- its a high-limit flowrate setpoint. please adjust. i am also confused why you would be controlling to high limit; bypass valves are usually for low-limit flows (that is, to keep the flow in the primary loop above a certain minimum value). what is the control strategy here?

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 5]
```diff
@@ -885,6 +895,17 @@ MINFC:
   - bypass_valve_percentage_command
   - min_flowrate_setpoint
   - flowrate_sensor
+  
+MAXFC:
+  id: "16039155144679489536"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:900]: you do not need to add IDs, that is done automatically by our system.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 6]
```diff
@@ -29,6 +29,48 @@ VAV_SD_CSP:
   - VAV
   - SD
   - CSP
+  
+VAV_PDSCV:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:33]: what does it control to? this seems like an incomplete VAV.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 7]
```diff
@@ -29,6 +29,48 @@ VAV_SD_CSP:
...
+  description: "Pressure-dependent supply damper control."
+  is_canonical: true
+  implements:
+  - VAV
+  - PDSCV
+
+VAV_SFM:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:40]: what does it control to? this seems like an incomplete VAV.

File: ontology/yaml/resources/HVAC/entity_types/VAV.yaml [hunk 8]
```diff
@@ -29,6 +29,48 @@ VAV_SD_CSP:
...
+  description: "Supply air flow monitoring for VAV. "
+  is_canonical: true
+  implements:
+  - VAV
+  - SFM  
+
+VAV_MINFC:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/VAV.yaml:47]: what does it control to? this seems like an incomplete VAV.

**Review by tasodorff — CHANGES_REQUESTED** [review]
Require major changes to VAV definitions. We want to reuse existing VAV types as much as possible, even if your translations to them are incomplete (if the devices don't send you all the data). Model the devices based on what they do, not just based on what data they send us.

**Comment by tasodorff** [comment]
Make the changes already requested, and ensure the validator tests pass.

**Comment by RathishKJ** [comment]
Closing the pull request since we have found an existing canonical in the Git...

**Pull request closed without merging**